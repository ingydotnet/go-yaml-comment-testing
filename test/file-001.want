kind: Document
content:
    - kind: Mapping
      content:
        - kind: Scalar
          head: |-
            #   comment 1
            # comment 2
          text: foo
        - kind: Scalar
          style: Double
          line: '#  comment 3'
          text: '1 '
        - kind: Scalar
          head: |
            #     comment 4
          foot: '#   comment 5'
          text: bar
        - kind: Scalar
          tag: '!!int'
          text: "2"
