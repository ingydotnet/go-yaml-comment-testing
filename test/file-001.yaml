#   comment 1
---
# comment 2
foo: 1   #  comment 3

  #     comment 4

bar: 2
  #   comment 5
