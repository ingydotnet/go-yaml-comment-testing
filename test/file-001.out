Kind: Document
More:  
    Kind: Mapping
    More:  
        Kind: Scalar
        Head:
          "#   comment 1
          # comment 2"
        Text: "foo"
        
        Kind: Scalar
        Styl: Double
        Line: "#  comment 3"
        Text: "1 "
        
        Kind: Scalar
        Head:
          "#     comment 4
          "
        Foot: "#   comment 5"
        Text: "bar"
        
        Kind: Scalar
        Tag : !!int
        Text: "2"
        
